### PR TITLE
User/v jandrew/revised enterprise policy

### PR DESCRIPTION
**Pull request by JasonAndrewWriter**

### CODE REVIEW

**Review by Bhuvana-Priya — APPROVED** [review]
Reviewed

**Comment by JasonAndrewWriter** [comment]
@mikehoffms could you review?

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 1]
```diff
@@ -35,13 +35,13 @@ To help you plan, consider the following schedule for Microsoft Partner Center a
 We are aware that Chromium has revised the timelines for Manifest V2 sunset. We will independently decide on MV3 migration timelines for Microsoft Edge Add-ons and share an update here. We continue to analyze the concerns raised by the extension developers and explore the optimal path for Microsoft Edge Add-ons ecosystem. Meanwhile, please refer to the [Chromium timelines](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/) for planning your extension's migration.
 
 
-| Timeframe	| Microsoft Partner Center changes | Microsoft Edge changes |
+| Timeframe    | Microsoft Partner Center changes | Microsoft Edge changes |
 |--- |--- |--- |
-| July 2022 (already in effect)	| Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
+| July 2022 (already in effect)    | Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:40]:
```suggestion
| July 2022 (already in effect) | Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
```

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 2]
```diff
@@ -35,13 +35,13 @@ To help you plan, consider the following schedule for Microsoft Partner Center a
...
 | TBD - See note above | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge using Enterprise policies. |
 | TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policy. |
 
-MV2 extensions will continue to be supported through enterprise policy at least until Chromium MV2 support timeline (which is currently January 2024).
+MV2 extensions will continue to be supported through the enterprise policy at least until Chromium MV2 support timeline (which is currently January 2024).
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:44]:
```suggestion
MV2 extensions will continue to be supported through the enterprise policy at least until Chromium Manifest V2 support timeline (which is currently January 2024). 
```

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 3]
```diff
@@ -35,13 +35,13 @@ To help you plan, consider the following schedule for Microsoft Partner Center a
 We are aware that Chromium has revised the timelines for Manifest V2 sunset. We will independently decide on MV3 migration timelines for Microsoft Edge Add-ons and share an update here. We continue to analyze the concerns raised by the extension developers and explore the optimal path for Microsoft Edge Add-ons ecosystem. Meanwhile, please refer to the [Chromium timelines](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/) for planning your extension's migration.
 
 
-| Timeframe	| Microsoft Partner Center changes | Microsoft Edge changes |
+| Timeframe    | Microsoft Partner Center changes | Microsoft Edge changes |
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:38]:
```suggestion
| Timeframe | Microsoft Partner Center changes | Microsoft Edge changes |
```

File: microsoft-edge/extensions-chromium/publish/contact-extensions-team.md [hunk 4]
```diff
@@ -28,4 +28,4 @@ You can contact the Microsoft Edge extensions support team request help, submit
 <!-- ====================================================================== -->
 ## Report copyright or trademark infringement
 
-If you think an item in the Microsoft Edge Add-ons website violates a copyright or trademark, please fill out the [Reporting Infringement](https://www.microsoft.com/en-us/concern/dmca) form. While filling up the Step-3 of the form in the **Product/Service/App** option, ensure you select **‘Microsoft Store on Windows’**. The Microsoft Edge extensions team will review your report and then take the necessary action.
\ No newline at end of file
+If you think an item in the Microsoft Edge Add-ons website violates a copyright or trademark, please complete the [Reporting Infringement](https://www.microsoft.com/en-us/concern/dmca) form. When finishing Step-3 of the form in the **Product/Service/App** option, ensure you select **'Microsoft Store on Windows'**. The Microsoft Edge extensions team will review your report and then take the necessary action.
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/publish/contact-extensions-team.md:31]: 'please' has 37 hits in 27 files.  I think it fits here b/c we're inviting user to go out of their way and do a favor.
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please - "Avoid please except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation."
```suggestion
If you think an item in the Microsoft Edge Add-ons website violates a copyright or trademark, please complete the [Reporting Infringement](https://www.microsoft.com/en-us/concern/dmca) form. When finishing Step 3 of the form in the **Product/Service/App** option, make sure to select **'Microsoft Store on Windows'**. The Microsoft Edge extensions team will review your report and then take the necessary action.
```

**Review by mikehoffms — CHANGES_REQUESTED** [review]
reviewed

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 5]
```diff
@@ -35,13 +35,13 @@ To help you plan, consider the following schedule for Microsoft Partner Center a
...
 |--- |--- |--- |
-| July 2022 (already in effect)	| Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
+| July 2022 (already in effect) | Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
 | TBD - See note above | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge using Enterprise policies. |
 | TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policy. |
 
-MV2 extensions will continue to be supported through enterprise policy at least until Chromium MV2 support timeline (which is currently January 2024).
+Manifest V2 extensions will continue to be supported through the enterprise policy at least until Chromium Manifest V2 support timeline (which is currently January 2024).
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:44]:
```suggestion
Manifest V2 extensions will continue to be supported through Enterprise policies at least until Chromium Manifest V2 support timeline (which is currently January 2024). 
```

File: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md [hunk 6]
```diff
@@ -35,13 +35,13 @@ To help you plan, consider the following schedule for Microsoft Partner Center a
...
 
 
-| Timeframe	| Microsoft Partner Center changes | Microsoft Edge changes |
+| Timeframe | Microsoft Partner Center changes | Microsoft Edge changes |
 |--- |--- |--- |
-| July 2022 (already in effect)	| Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
+| July 2022 (already in effect) | Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
 | TBD - See note above | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge using Enterprise policies. |
 | TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policy. |
```
Comment (mikehoffms) [inline microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md:42]:
```suggestion
| TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policies. |
```

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**